assets/app.js

```diff
@@ -0,0 +1,212 @@
+
+
+function removeActive() {
+  garden.classList.remove('btnActiv');
+  lawnBtn.classList.remove('btnActiv');
+  document.querySelector('#planting').classList.remove('btnActiv');
+}
+
+
+garden.addEventListener('click', function () {
+  removeActive();
+  service__card.forEach(
+    function (el) {
+      el.classList.remove('blur');
+    })
+  lawn.classList.add('blur');
+  planting.forEach(
+    function (el) {
+      el.classList.add('blur');
+    }
+  )
+  garden.classList.add('btnActiv');
+})
+
+
+lawnBtn.addEventListener('click', function () {
+  removeActive();
+  service__card.forEach(
+    function (el) {
+      el.classList.remove('blur');
+    })
+  planting.forEach(
+    function (el) {
+      el.classList.add('blur');
+    }
+  );
+  gardens.forEach(
+    function (el) {
+      el.classList.add('blur');
+    }
+  )
+  lawnBtn.classList.add('btnActiv');
+})
+
+
+document.querySelector('#planting').addEventListener('click', function () {
+  removeActive();
+  service__card.forEach(
+    function (el) {
+      el.classList.remove('blur');
+    })
+  lawn.classList.add('blur');
+  gardens.forEach(
+    function (el) {
+      el.classList.add('blur');
+    }
+  )
+  document.querySelector('#planting').classList.add('btnActiv');
+})
+
+// ................./////////////////////////////////////////////////////
+
+const basicPrice = document.getElementById('pricesBasics');
+const standardPrice = document.getElementById('pricesStandard');
+const proPrice = document.getElementById('pricesPro');
+
+
+
+function removeActiveClasses(btn) {
+  btn.classList.remove('active');
+  btn.nextElementSibling.classList.remove('active');
+}
+
+function closeMenus(targetId) {
+  const otherBtns = document.querySelectorAll('.prices__btn:not(#' + targetId + ')');
+  otherBtns.forEach(btn => {
+    removeActiveClasses(btn);
+  });
+}
+
+function handleMenu(btn) {
+  closeMenus(btn.getAttribute('id'));
+  if (btn.classList.contains('active')) {
+    removeActiveClasses(btn);
+  } else {
+    btn.classList.add('active');
+    btn.nextElementSibling.classList.add('active');
+  }
+}
+
+function isContentClicked(el) {
+  return !el.classList.contains("prices__btn")
+    && !el.classList.contains("prices-menu")
+    && !el.parentElement.classList.contains("prices-menu")
+}
+
+document.addEventListener('click', function (event) {
+  if (isContentClicked(event.target)) {
+    const activeBtns = document.querySelectorAll('.prices__btn.active');
+    if (activeBtns.length > 0) {
+      activeBtns.forEach(btn => {
+        removeActiveClasses(btn);
+      });
+    }
+  }
+});
+
+
+/////////////////////////////////////////////////////////////////////////////
+
+const menuCity = document.querySelector('#city-list');
+const cityMenuItems = document.querySelectorAll(".city-item");
+const cityBtn = document.querySelector('.contacts__btn');
+
+
+function openCityMenu() {
+  if (menuCity.classList.contains('city-list-active')) {

```

Comment:
ДЛя полной красоты можно вынести `city-list-active` в отдельную константу (объявить выше const CITY_ACTIVE_CLASS = 'city-list-active') и использовать её

```suggestion
 function openCityMenu() {
 if (menuCity.classList.contains(CITY_ACTIVE_CLASS)) {
    menuCity.classList.remove(CITY_ACTIVE_CLASS);
  } else {
    menuCity.classList.add(CITY_ACTIVE_CLASS);
  }
}
```